go.mod

```diff
@@ -14,6 +16,8 @@ require (
 	github.com/stretchr/testify v1.5.1
 	golang.org/x/sync v0.0.0-20190911185100-cd5d95a43a6e
 	golang.org/x/sys v0.0.0-20200519105757-fe76b779f299
+	google.golang.org/grpc v1.30.0
+	google.golang.org/protobuf v1.23.0
```

Comment:
Why is the protobuf imported twice?

Reply:
They're both different packages. Grpc is the framework used and protobuf is how data serialization is done. Similar to SOAP and XML.

Reply:
So both the packages have been imported by the auto generated code which we don't control. One is being used for the proto files and other for the protocol itself. At some point, the compiler will start using google.golang.org/protobuf , I suppose as I read this: https://developers.google.com/protocol-buffers/docs/reference/go/faq